src/pages/index.tsx

```diff
@@ -139,11 +139,11 @@ const HomePage = (props: IHomePage) => {
         </PageSection>
       )}
       <PageSection>
-        <Contents articles={props.articles?.slice(0,3)} friends={props.friends} />
-      </PageSection>
-      <PageSection size="full" styles={{ backgroundColor: color.background.black }}>
-        <Download />
+        <Contents articles={props.articles?.slice(0, 3)} friends={props.friends} />
       </PageSection>
+      {props.activeTags && props.activeTags.length > 0 &&
```

Comment:
Chyba opakowałbym to w osobny komponent pt. ActiveTags czy coś takiego. Może się rozrosnąć